configs/seL4Config.cmake

```diff
@@ -185,7 +156,6 @@ unset(KernelArmCPU CACHE)
 unset(KernelArmArmV CACHE)
 
 # Blacklist platforms without MCS support
```

Comment:
maybe this comment should also be removed. Or updated to say that platform must explicitly turn this off now.

Reply:
I think your other PR that updates the comment should be sufficient.  I haven't changed the behavior of KernelPlatformSupportsMCS in a way that makes the comment wrong. 
KernelPlatformSupportsMCS still works the same way:
- It's default to ON, but platforms that don't support MCS set it to off.
What's changed is that the result isn't written into the cache which was requiring the entry to be deleted each time the script reruns.